tools/docs/config.ts

```diff
@@ -72,15 +76,13 @@ export async function generateConfig(dist: string, bot = false): Promise<void> {
       if (headerIndex === -1) {
         headerIndex = configOptionsRaw.indexOf(`### ${option.name}`);
       }
-      if (bot) {
```

Comment:
I changed logic here,  i added cli and env for both Self hosted configuration and normal configuration, is that ok?